realm/realm-library/src/androidTest/java/io/realm/RealmMigrationTests.java

```diff
@@ -694,9 +694,24 @@ public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
     public void openPreNullWithRequired() throws IOException {
         configFactory.copyRealmFromAssets(context,
                 "default-before-migration.realm", Realm.DEFAULT_REALM_NAME);
+        RealmMigration migration = new RealmMigration() {
```

Comment:
It is a bit annoy ... but I think it is better to update the `default-before-migration.realm` instead. This might have side effect on the test case itself it make the test case gets confused ... YEAH, update the binary file is really annoy.